Update config missed in last GSI update

WalterKolczynski-NOAA opened the pull request:
When the GSI version was updated in PR #530, updates to the config.anal file were mistakenly omitted. That file is now updated following ops (PR #489).

The indentation of the config.anal file is also fixed.

Refs: PR #530

WalterKolczynski-NOAA committed:
Update GSI config missed in PR #530

When the GSI version was updated in PR #530, updates to the config.anal
file were mistakenly omitted. That file is now updated following ops
(PR #489).

Refs: PR #530

WalterKolczynski-NOAA commented:
I haven't tested these changes, since they are straightforward and just a copy of what was done in ops. But I will if requested.

KateFriedman-NOAA reviewed (approved):
Looks good, thanks for getting this in @WalterKolczynski-NOAA ! My bad for missing!

CatherineThomas-NOAA reviewed (changes requested):
Thanks @WalterKolczynski-NOAA for acting quickly on this.  I don't think it needs to be cycled for testing.

The v16.1.5 changes look good but the v16.1.4 changes are still missing.  This was originally pulled into the operations branch in PR #451.  This could either be added to this PR or if it's easier to submit a new PR, that's fine with me, too.

WalterKolczynski-NOAA committed:
Add GSI 16.1.4 updates to conf, fix indentation

More settings missed in PR #530. Also fixed indentation better so
that all indentations use tabs (it was mixed).

Refs: PR #530

CatherineThomas-NOAA commented on parm/config/config.anal:
```diff
@@ -62,81 +62,116 @@ export OBERROR=$FIXgsi/prepobs_errtable.global
+	#   Assimilate 135 (T) & 235 (uv) Canadian AMDAR observations
+	if [[ "$CDATE" -ge "2020040718" && "$CDATE" -lt "2020052612" ]]; then
+		export CONVINFO=$FIXgsi/gfsv16_historical/global_convinfo.txt.2020040718
+		export OBERROR=$FIXgsi/gfsv16_historical/prepobs_errtable.global.202004071
```
The date got cut off at the end of this file.  It should be 2020040718

WalterKolczynski-NOAA replied:
Good catch. Fixed.

WalterKolczynski-NOAA committed:
Fix date typo in config.anal

CatherineThomas-NOAA reviewed (approved):
Looks good.  Thanks @WalterKolczynski-NOAA!
